### ISSUE TITLE
Be aware of util API change

### ISSUE THREAD

**Issue by peduarte** [hauptrolle/stitches-utils#8]
In the next release, the util API will change to:

```js
utils: {
  mx: (value, config) => ({})
}
```

PR: https://github.com/modulz/stitches/pull/209

**Comment by braposo** [comment]
Yes I mentioned it in [my comment](https://github.com/hauptrolle/stitches-utils/pull/2#issuecomment-690777947) but wasn't sure if that had landed already or not. Do you know when will that be released @peduarte?

**Comment by peduarte** [comment]
Not yet, but I'll post it on Discord 🔥

**Comment by peduarte** [comment]
Canary Release:  `0.0.3-canary.1` :tada:

```bash
yarn add @stitches/react@0.0.3-canary.1
```

Fixes and Improvements:
- TS Typing improvement https://github.com/modulz/stitches/pull/166
- text-decoration shorthand https://github.com/modulz/stitches/pull/159
- Fix check against Deno  https://github.com/modulz/stitches/pull/201
- Add CodeSandbox CI https://github.com/modulz/stitches/pull/198
- Fix bug with missing closing curly in keyframes https://github.com/modulz/stitches/pull/202
- Strict mode https://github.com/modulz/stitches/pull/206
- Fix Variant typing https://github.com/modulz/stitches/pull/218

:boom: Breaking:
- Simplify Util api  https://github.com/modulz/stitches/pull/209
- Fix global and keyframes style injection https://github.com/modulz/stitches/pull/208

Demo: https://codesandbox.io/s/stitches-003-canary1-vdudm
